projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -304,6 +317,18 @@ export const getStepContent = ( state, stepSlug ) => {
 				ctaText: __( 'Install Creative Mail', 'jetpack' ),
 				illustration: 'assistant-creative-mail',
 			};
+		case 'newsletter':
+			return {
+				progressValue: '70',
+				question: __( 'Send subscribers your latest blog posts via email?', 'jetpack' ),
+				description: __(
+					'With Jetpack Newsletter you can keep your audience engaged by automatically sending your content via email. <ExternalLink>Learn more</ExternalLink>',
+					'jetpack'
+				),
+				descriptionLink: 'https://jetpack.com/newsletter/',
```

Comment:
`https://jetpack.com/newsletter/` doesn't lead anywhere right now, but it feels like it should!

Once we have that link working, we could replace the direct link by a tracked link here:

```suggestion
				descriptionLink: getRedirectUrl( 'jetpack-newsletter-landing' ),
```

Reply:
`https://jetpack.com/newsletter/` will be the landing page when it's live 

> Once we have that link working, we could replace the direct link by a tracked link here: 

We could, but for now since there's other links on that page which aren't redirected let's sort all of them in another PR.

Reply:
Done in #31996.